spyder/preferences/languageserver.py

```diff
@@ -608,30 +607,356 @@ def mouseDoubleClickEvent(self, event):
+        code_style_g_layout.addWidget(code_style_select.label, 3, 0)
+        code_style_g_layout.addWidget(code_style_select.textbox, 3, 1)
+        code_style_g_layout.addWidget(code_style_ignore.label, 4, 0)
+        code_style_g_layout.addWidget(code_style_ignore.textbox, 4, 1)
+        code_style_g_layout.addWidget(code_style_max_line_length.plabel, 5, 0)
+        code_style_g_layout.addWidget(
+            code_style_max_line_length.spinbox, 5, 1)
+
+        # Set Code style options enabled/disabled
+        code_style_g_widget = QWidget()
+        code_style_g_widget.setLayout(code_style_g_layout)
+        code_style_g_widget.setEnabled(self.get_option('pycodestyle'))
+        code_style_check.toggled.connect(code_style_g_widget.setEnabled)
+
+        # Code style layout
+        code_style_layout = QVBoxLayout()
+        code_style_layout.addWidget(code_style_label)
+        code_style_layout.addWidget(code_style_check)
+        code_style_layout.addWidget(code_style_g_widget)
+
+        code_style_widget = QWidget()
+        code_style_widget.setLayout(code_style_layout)
+
+        # --- Docstring tab ---
+        # Docstring style label
+        numpy_url = (
+            "<a href='https://numpydoc.readthedocs.io/en/"
+            "latest/format.html'>Numpy</a>")
+        pep257_url = (
+            "<a href='https://www.python.org/dev/peps/pep-0257/'>Pep 257</a>")
+        docstring_style_codes = _(
+            "<a href='http://www.pydocstyle.org/en/stable"
+            "/error_codes.html'>page</a>")
+        docstring_style_label = QLabel(
+            _("Here you can decide if you want to perform style analysis on "
+              "your docstrings according to the {} or {} conventions. You can "
+              "also decide if you want to show or ignore specific errors, "
+              "according to the codes found on this {}.").format(
+                  numpy_url, pep257_url, docstring_style_codes))
+        docstring_style_label.setOpenExternalLinks(True)
+        docstring_style_label.setWordWrap(True)
+
+        # Docstring style checkbox
+        docstring_style_check = self.create_checkbox(
+            _("Enable docstring style linting"),
+            'pydocstyle')
+
+        # Docstring style options
+        docstring_style_convention = self.create_combobox(
+            _("Choose the convention used to lint docstrings: "),
+            (("Numpy", 'numpy'),
+             ("Pep 257", 'pep257')),
+            'pydocstyle/convention')
+        docstring_style_add_select = self.create_lineedit(
+            _("Display the following errors in addition to the "
+              "specified convention:"),
+            'pydocstyle/add_select', alignment=Qt.Horizontal, word_wrap=False,
+            placeholder=_("Example codes: D100, D200"))
+        docstring_style_add_ignore = self.create_lineedit(
+            _("Ignore the following errors in addition to "
```

Comment:
``Ignore the following errors:``

Reply:
But then how we can differenciate between this option and the one below? (add_ignore vs ignore or add_select vs select)?

Reply:
See my [comment](https://github.com/spyder-ide/spyder/pull/8647#discussion_r275167990) above, which replaces the two sets of fields with one, controlled by the dropdown.

Reply:
Oh didn't see that, I will check :+1:

Reply:
Discussed above.